src/lib/domain/ng-package-format.ts

```diff
@@ -163,6 +168,15 @@ export class NgEntryPoint {
     }
   }
 
+  /**
+   * TODO
+   * Dependencies of this entry point on other entry points in the package.
+   * List of module IDs
+   */
+  public dependendingOnEntryPoints: string[] = [];
+
+  public buildStatus: 'failed'|'success'|'pending'|'inprogress' = 'pending';
```

Comment:
My i suggest to use string enums rather than types?

enum BuildStatus {
    InProgress = ‘InProgress’
}

Usage would be ‘BuildStatus.InProgress’ 

Unless you want to use normal enums.

Sry can only write messages, as I am in a hurry!